pandas/core/generic.py

```diff
@@ -4285,6 +4307,31 @@ def ftypes(self):
         """
         Return the ftypes (indication of sparse/dense and dtype)
         in this object.
+
+        Notes
+        -----
+        Sparse data should have the same dtypes as its dense representation
+
+        See Also
+        --------
+        dtypes, SparseDataFrame
```

Comment:
Instead of simply `dtypes` you should use `pandas.DataFrame.dtypes`. You can even try linking to dtypes with the notation found in the guide: :meth:\`pandas.Series.sum\`